pkg/tfbridge/provider.go

```diff
@@ -1360,9 +1361,11 @@ func (p *Provider) Read(ctx context.Context, req *pulumirpc.ReadRequest) (*pulum
 		return nil, errors.Wrapf(err, "unmarshaling %s's instance state", urn)
 	}
 
+	var isImport bool
 	// If we are in a "get" rather than a "refresh", we should call the Terraform importer, if one is defined.
 	isRefresh := len(req.GetProperties().GetFields()) != 0
 	if !isRefresh && res.TF.Importer() != nil {
+		isImport = true
```

Comment:
This is confusing, we need sort out how Resource.get() method maps to the Read call here, as things stand I isImport value might be true during a Resource.get(). I can't think of this being particularly harmful but it is confusing.